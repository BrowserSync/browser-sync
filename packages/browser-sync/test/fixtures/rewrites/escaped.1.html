<a href="http://demo.l5"></a>
phpdebugbar.setOpenHandler(new
PhpDebugBar.OpenHandler({"url":"http:\/\/demo.l5\/_debugbar\/open"}));
