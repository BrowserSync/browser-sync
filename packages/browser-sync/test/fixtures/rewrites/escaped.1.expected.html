<a href="http://192.168.0.4:3002"></a>
phpdebugbar.setOpenHandler(new PhpDebugBar.OpenHandler({"url":"http:\/\/192.168.0.4:3002\/_debugbar\/open"}));
